tests/transactions/tests.py

```diff
@@ -426,3 +428,15 @@ def test_atomic_does_not_leak_savepoints_on_failure(self):
 
                 # This is expected to fail because the savepoint no longer exists.
                 connection.savepoint_rollback(sid)
+
+    @skipIf(connection.features.autocommits_when_autocommit_is_off,
+            "This test requires a non-autocommit mode that doesn't autocommit.")
+    def test_orm_query_without_autocommit(self):
+        # Regression test for #24921
```

Comment:
If possible to use a more descriptive comment and only refer to the ticket if it provides additional background information, that's preferred from my perspective.